module/Entra/AdditionalFunctions/Remove-EntraAdministrativeUnitMember.ps1

```diff
@@ -0,0 +1,82 @@
+function Remove-EntraAdministrativeUnitMember {
```

Comment:
Add license information at the top of this file

Reply:
done